pkg/profileparser/profileparser.go

```diff
@@ -814,6 +814,10 @@ func newStandardParser() *referenceParser {
 	if err != nil {
 		log.Error(err, "Could not register NIST-800-53 reference parser") // not much we can do here..
 	}
+	bsiperr := p.registerStandard("BSI", `^https://www\.bsi\.bund\.de/SharedDocs/Downloads/EN/BSI/Grundschutz/International/bsi_it_gs_comp_2022\.pdf$`)
```

Comment:
I'd like to find a way where we can unit test these so we can protect against regressions in the future.

Doesn't look like we've tested this in the past 😞 - something to look into though.